### PR TITLE
[FIX] Fixing use of pc. in light.js

### PR DESCRIPTION
**Pull request by mvaligursky**

### CODE REVIEW

**Review by willeastcott — APPROVED** [review]

File: src/scene/light.js [hunk 1]
```diff
@@ -326,15 +332,15 @@ class Light {
...
                 height: 64,
                 format: format,
-                addressU: pc.ADDRESS_CLAMP_TO_EDGE,
-                addressV: pc.ADDRESS_CLAMP_TO_EDGE,
-                type: pc.TEXTURETYPE_RGBE,
-                magFilter: pc.FILTER_LINEAR,
-                minFilter: pc.FILTER_NEAREST,
+                addressU: ADDRESS_CLAMP_TO_EDGE,
+                addressV: ADDRESS_CLAMP_TO_EDGE,
+                type: TEXTURETYPE_DEFAULT,
```
Comment (willeastcott) [inline src/scene/light.js:341]: This is changing more than removing the `pc.`. What's the justification here?

Comment (mvaligursky) [reply src/scene/light.js:341]: A bug I introduced by copy & paste when writing this code.

Comment (willeastcott) [reply src/scene/light.js:341]: OK, so it's correct now? Cool.

Comment (mvaligursky) [reply src/scene/light.js:341]: yes it's just normal RGBA texture without a special meaning for the alpha.

**Pull request merged**